kernel/Makefile

```diff
@@ -62,6 +81,23 @@ $(info -- KernelSU Manager signature hash: $(KSU_EXPECTED_HASH))
 ccflags-y += -DEXPECTED_SIZE=$(KSU_EXPECTED_SIZE)
 ccflags-y += -DEXPECTED_HASH=\"$(KSU_EXPECTED_HASH)\"
 
+CLOSE_PARENTHESES = )
+ifeq ($(shell grep -q "int path_umount" $(srctree)/fs/namespace.c; echo $$?),0)
+ccflags-y += -DKSU_UMOUNT
+else
+$(info --  __________________________________)
+$(info -- / Did you know you can backport    \)
+$(info -- | path_umount to get module umount |)
+$(info -- \ working?                         /)
+$(info --  ----------------------------------)
+$(info --         \   ^__^)
+$(info --          \  (oo)\_______)
+$(info --             (__)\       $(CLOSE_PARENTHESES)\/\)
+$(info --                 ||----w |)
+$(info --                 ||     ||)
+$(info -- Read: https://kernelsu.org/guide/how-to-integrate-for-non-gki.html#path_umount)
```

Comment:
I don't want to open this file every time and see that most of the content is useless to me. Keep it simple please, one line is enough.